### PR TITLE
Longer error message for curl_{easy,multi}_setopt 48

### PR DESCRIPTION
**Pull request by StefanKarpinski**
Slightly passive aggressive, but I'm tired of people filing issues about their misconfigured Julia installs.

### CODE REVIEW

**Review by DilumAluthge — APPROVED** [review]

File: src/Curl/utils.jl [hunk 1]
```diff
@@ -24,10 +24,54 @@ macro check(ex::Expr)
...
+
+                  1. You are using an unofficial Julia build which is \
+                  configured to use a system libcurl library that is not \
+                  recent enough; you may be able to fix this by upgrading the \
+                  system libcurl. You should complain to your distro \
+                  maintainers for allowing Julia to use a too-old libcurl \
+                  version and consider using official Julia binaries instead.
+
+                  2. You are overriding the library load path by setting \
+                  `LD_LIBRARY_PATH`, in which case I'm afraid you are in \
```
Comment (KristofferC) [inline src/Curl/utils.jl:54]: "I'm afraid" sounds weird in an error message imo.

Comment (StefanKarpinski) [reply src/Curl/utils.jl]: Yeah, I'll remove that.

**Comment by StefanKarpinski** [comment]
Any clues why force pushing this didn't kick of CI again?

**Comment by giordano** [comment]
https://www.githubstatus.com/incidents/y5hdmv0p49x3

**Pull request merged**